Userland/Libraries/LibWeb/Layout/GridFormattingContext.cpp

```diff
@@ -1720,7 +1720,47 @@ void GridFormattingContext::resolve_track_spacing(GridDimension const dimension)
             track.base_size = space_between_tracks;
         }
     } else {
```

Comment:
could we unify handling of justify-content and align-content properties within this function? then and else blocks in resolve_track_spacing looks basically the same except using columns or rows based on property name.